### DISCUSSION TITLE
GeoJsonLayer line customization

### DISCUSSION THREAD

**Discussion by hifichevymane** [visgl/deck.gl#8879] [Q&A] [answered]
Hey, I want to customize my `GeoJsonLayer` lines style. I want them to be dashed instead of solid. Is there some possibility to do it?

**Comment by donmccurdy** [comment, marked as answer]
Hi @hifichevymane — I believe you'll want to use [PathStyleExtension](https://deck.gl/docs/api-reference/extensions/path-style-extension) to apply dashed line styles.

**Reply by hifichevymane** [reply]
Thank you! That's what I was searching
